code.json

```diff
@@ -84,6 +84,50 @@
       "email": "jlaura@usgs.gov"
     },
 
+    "date": {
+      "metadataLastUpdated": "2023-04-19"
+    }
+  },{
+    "name": "ISIS3",
+    "organization": "U.S. Geological Survey",
+    "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
+    "version": "8.1.0_RC1",
+    "status": "Production",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/astrogeology/isis/-/raw/7.1.0/LICENSE.md"
```

Comment:
This should be changed to the right version

Reply:
Looks like the above code entry URLs haven't been updated either, I can change that

Reply:
Sure, I think that would be a good idea